onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -5570,12 +5570,15 @@ def _aten_native_batch_norm_inference_onnx(
         momentum=momentum,
         training_mode=training,
     )
+    # NOTE: mean and var are ommited in inference mode
     # Cannot return 2 dup output, so have to do twice with different variable name
     empty_mean = op.CastLike(op.Shape(input, start=0, end=0), norm)
     empty_var = op.CastLike(op.Shape(input, start=0, end=0), norm)
     return norm, empty_mean, empty_var
 
 
+# TODO: This op is using duplicated code from aten_native_batch_norm,
```

Comment:
Prefer adding an issue link?

Reply:
https://github.com/microsoft/onnxscript/issues/1125

Reply:
Done in comments also